Shared/Mirror/mirrorClient.js

```diff
@@ -0,0 +1,314 @@
+"use strict";
```

Comment:
Missing file headers with copyright notification, creator and optional short description of the script, see other scripts in repo for examples.